tests/yam/migration/migration_auto.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Migration activation then reboot to perform migration.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use registration 'register_addons_cmd';
+use Utils::Backends 'is_pvm';
+use utils qw(reconnect_mgmt_console zypper_call);
+use registration 'scc_version';
+
+sub run {
+    my ($self) = @_;
+    my $repo_home = "http://download.suse.de/ibs/home:/fcrozat:/SLES16/SLE_$releasever";
+    my $repo_images = 'http://download.suse.de/ibs/home:/fcrozat:/SLES16/images/';
```

Comment:
better put these two lines after select_console, take into account that this block is temporary, so easy to drop later, not a fixed initialization that would stay in the future.